sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/aio/_path_client_async.py

```diff
@@ -0,0 +1,500 @@
+        :keyword ~azure.core.MatchConditions match_condition:
+            The match condition to use upon the etag.
+        :keyword int timeout:
+            The timeout parameter is expressed in seconds.
+        :return: Dict[str, Union[str, datetime]]
+        """
+        options = self._create_path_options(
+            resource_type,
+            content_settings=content_settings,
+            metadata=metadata,
+            **kwargs)
+        try:
+            return await self._client.path.create(**options)
+        except StorageErrorException as error:
+            process_storage_error(error)
+
+    async def _delete(self, **kwargs):
+        # type: (bool, **Any) -> None
+        """
+        Marks the specified path for deletion.
+
+        :keyword lease:
+            Required if the file/directory has an active lease. Value can be a LeaseClient object
+            or the lease ID as a string.
+        :type lease: ~azure.storage.filedatalake.DataLakeLeaseClient or str
+        :param ~datetime.datetime if_modified_since:
+            A DateTime value. Azure expects the date value passed in to be UTC.
+            If timezone is included, any non-UTC datetimes will be converted to UTC.
+            If a date is passed in without timezone info, it is assumed to be UTC.
+            Specify this header to perform the operation only
+            if the resource has been modified since the specified time.
+        :param ~datetime.datetime if_unmodified_since:
+            A DateTime value. Azure expects the date value passed in to be UTC.
+            If timezone is included, any non-UTC datetimes will be converted to UTC.
+            If a date is passed in without timezone info, it is assumed to be UTC.
+            Specify this header to perform the operation only if
+            the resource has not been modified since the specified date/time.
+        :keyword str etag:
+            An ETag value, or the wildcard character (*). Used to check if the resource has changed,
+            and act according to the condition specified by the `match_condition` parameter.
+        :keyword ~azure.core.MatchConditions match_condition:
+            The match condition to use upon the etag.
+        :param int timeout:
+            The timeout parameter is expressed in seconds.
+        :return: None
+        """
+        options = self._delete_path_options(**kwargs)
+        try:
+            return await self._client.path.delete(**options)
+        except StorageErrorException as error:
+            process_storage_error(error)
+
+    async def set_access_control(self, owner=None,  # type: Optional[str]
+                           group=None,  # type: Optional[str]
+                           permissions=None,  # type: Optional[str]
+                           acl=None,  # type: Optional[str]
+                           **kwargs):
+        # type: (...) -> Dict[str, Union[str, datetime]]
```

Comment:
Most of these arguments should be made keyword only I think

Reply:
we cannot set access control with empty parameter, while I'm not sure if it is a service bug. I will check with FE people